www/_posts/2024/2024-11-21-outstanding-award-nominations.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post.html
+title: "Nominate a PyLady for the Outstanding PyLadies Award!"
+tags: [Global, "Outstanding PyLady Award"]
+author: Bethany Garcia
+---
+
+_Reminder: [nominate a PyLady][this form] by the end of the year for the Oustanding PyLady Award!_
+
+**Attention all PyLadies and PyLadies Supporters!**
+
+The **December 31st, 2024** (AOE) deadline is fast approaching for Outstanding PyLady Award nominations!
+
+<figure>
+    <img src="{{ get_asset('images/blog/opa_badge_pin.png') }}" alt="Gold lanyard pin with 'Outstanding PyLady Award Recipient' written surrounding the PyLady geek logo" width="50%" style="margin-left: auto; margin-right: auto; margin-bottom: 2em; margin-top: 2em; display: block;">
+<figcaption style="text-align: center;"><i>Outstanding PyLady Award recipient lanyard pin. Photo taken by <a href="https://x.com/psobot">@psobot</a>.</i></figcaption></figure>
+
+The PyLadies community is made up of many remarkable members and volunteers that make significant 
+contributions to bettering the Python community. The Outstanding PyLady award is meant to elevate 
+diverse Pythonistas doing amazing things, highlighting their achievements, and thanking them for 
+all the work they've contributed.
+
+Nominations are accepted year-round, but December 31st is the cut-off date for recognition at 
+PyCon US 2025. Recognition will take the form of an award certificate, plus **one** of the following:
+
+* Free registration and travel assistance (_if needed_) to PyCon US 2025 <sup>1</sup>, or
+* A cash award of $1000 USD <sup>2</sup>.
+
+Self nominations and multiple nominations are accepted (_and encouraged!_), as are nominations
+from outside the PyLadies community; just fill out [this form] per nominee. Winners will be
+notified by the Global Council.
+
+Let us know about all the wonderful work that's been done this year and get those nominations in!
+
+To learn more about the process or see who's previously received the award, checkout the 
+[PyLadies kit](https://kit.pyladies.com/en/latest/global/award.html).
```

Comment:
This page has not been updated with previous award recipients. Could you all please update that?
Thank you.